lm_eval/tasks/tinyBenchmarks/README.md

```diff
@@ -0,0 +1,113 @@
+# tinyBenchmarks
+
+### Paper
+
+Title: `tinyBenchmarks: evaluating LLMs with fewer examples`
+
+Abstract: https://arxiv.org/abs/2402.14992
+
+The versatility of large language models (LLMs) led to the creation of diverse benchmarks that thoroughly test a variety of language models' abilities. These benchmarks consist of tens of thousands of examples making evaluation of LLMs very expensive. In this paper, we investigate strategies to reduce the number of evaluations needed to assess the performance of an LLM on several key benchmarks. For example, we show that to accurately estimate the performance of an LLM on MMLU, a popular multiple-choice QA benchmark consisting of 14K examples, it is sufficient to evaluate this LLM on 100 curated examples. We release evaluation tools and tiny versions of popular benchmarks: Open LLM Leaderboard, MMLU, HELM, and AlpacaEval 2.0. Our empirical analysis demonstrates that these tools and tiny benchmarks are sufficient to reliably and efficiently reproduce the original evaluation results. 
+
+Homepage: -
+
+All configs and utils mirror the ones from their original dataset!
+
+### Groups and Tasks
+
+#### Groups
+
+* `tinyBenchmarks`
+
+#### Tasks
+
+* `tinyArc`, `tinyGSM8k`, `tinyHellaswag`, `tinyMMLU`, `tinyTruthfulQA`, `tinyWinogrande`,
+
+### Usage
+
+*tinyBenchmarks* can evaluate different benchmarks with a fraction of their examples.
+To obtain accurate results, the score vectors obtained with this task require post-processing using the *tinyBenchmarks*-package (see [here](https://github.com/felipemaiapolo/tinyBenchmarks/blob/main/README.md?plain=1)).
+
+You can install our package by running the following commands on the terminal
+
+``` :sh
+$ pip install git+https://github.com/felipemaiapolo/tinyBenchmarks
+```
+
+Through the package, the ability parameter $\theta$ from the IRT model will be estimated using all the available data. For `benchmark='lb'` or `benchmark='helm_lite'`, the dimension of `y` should be 600 and 1000, respectively, where the correctness values must obey the following order 
+- For the Open LLM Leaderboard: TruthfulQA, GSM8K, Winogrande, ARC, HellaSwag, and MMLU;
+
+For all other, benchmarks the dimension of `y` should be 100.
+
+```python
+import json
+import tinyBenchmarks as tb
+import numpy as np
+
+# Choose benchmark (e.g. hellaswag)
+benchmark = 'hellaswag' # possible benchmarks:
+                        # ['lb','mmlu','alpaca','helm_lite','truthfulqa',
+                        #  'gsm8k', 'winogrande', 'arc', 'hellaswag']
+
+# Get score vector from output-file (the metric `acc_norm` depends on the benchmark)
+file_path = '<your-output-file.jsonl>'
+with open(file_path, 'r') as file:
+    outputs = json.load(file)
+y = np.array([float(item['acc_norm']) for item in outputs])
```

Comment:
can we add a sort by doc id in here so that it works for all data-parallel world sizes?